bigtable/admin.go

```diff
@@ -275,6 +278,35 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// UpdateTable updates a table in the instance from the given configuration.
+// Only deletion protection can be updated at this moment
+func (ac *AdminClient) UpdateTable(ctx context.Context, conf *TableConf) (updated bool, err error) {
+	if conf.TableID == "" {
+		return false, errors.New("TableID is required")
+	}
+	if conf.Families != nil || conf.SplitKeys != nil {
+		return false, errors.New("Only deletion protection field can be updated")
+	}
+
+	if conf.DeletionProtection != nil {
+		ctx = mergeOutgoingMetadata(ctx, ac.md)
+
+		updateMask := &field_mask.FieldMask{}
+		updateMask.Paths = append(updateMask.Paths, "deletion_protection")
+
+		req := &btapb.UpdateTableRequest{
+			Table: &btapb.Table{
+				Name:               conf.TableID,
+				DeletionProtection: *conf.DeletionProtection,
+			},
+			UpdateMask: updateMask,
+		}
+		_, err := ac.tClient.UpdateTable(ctx, req)
```

Comment:
Why are we ignoring the response here?

Reply:
+1

Also, if we get an error from the call to the service, should we return true to the caller?